floof.yaml

```
@@ -0,0 +1,59 @@
+default:
+  - command:
+      run: cargo run --bin export_schema -- ../frontend/build/schema.graphql
+      workdir: backend
+  - concurrently:
+    - http:
+        proxy: localhost:3080
+    - run-task: frontend
+    - run-task: backend
+
+backend:
+  - set-workdir: backend
+  - watch:
+      paths:
+        - Cargo.toml
+        - Cargo.lock
+        - src/
+      run:
+        # TODO: We only need to reexport schema if something in `src/api/` changed
+        - on-change: cargo run --bin export_schema -- ../frontend/build/schema.graphql
+        - reload:
+        - cargo run
+
+frontend:
+  - set-workdir: frontend
+  - watch:
+      paths:
+        - .babelrc
+        - .eslintrc.js
+        - package-lock.json
+        - package.json
+        - tsconfig.json
+        - webpack.config.js
+        - src/
+      run:
+        - npx relay-compiler
+        - npx tsc --skipLibCheck
+        - npx eslint src
```

```suggestion
        - npx eslint .
```

We want to lint everything